stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ApplicationDescriptor.java

```diff
@@ -61,8 +63,38 @@ protected void onGetChildren(Application element, Accumulator<Object> children)
         children.store(activity);
       }
     }
+    storeWindowIfNeeded(element,children,activities);
   }
+  private void storeWindowIfNeeded(Application application, Accumulator<Object> children, List<WeakReference<Activity>> activities){
+    List<View>windows = WindowFetcher.getWindowViews(application);
+    if (windows == null){
+      return;
+    }
+    for (View view : windows) {
+      if (belongToActivity(view, activities)) {
```

Comment:
nit: this logic reads better in reverse: `if (!belongToActivity(...)) { children.store(view); }`